docs/user_guide/migration.ipynb

```diff
@@ -0,0 +1,306 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Migration to momepy 1.0 API\n",
+    "\n",
+    "Starting with the version 0.8, momepy contains completely new API (and refactored internals) that will become the only API in the momepy 1.0. Given this is a complete reimplemnation of nearly entirety of the package, it is not compatible with the legacy class-based API. This notebook contains a succint migration guide, highlighiting the key differences between the two APIs and outlines required changes to existing code to make it compatible with upcoming 1.0 release."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import geopandas as gpd\n",
+    "from libpysal import graph, weights\n",
+    "\n",
+    "import momepy\n",
+    "\n",
+    "buildings = gpd.read_file(\n",
+    "    momepy.datasets.get_path(\"bubenec\"), layer=\"buildings\"\n",
+    ")\n",
+    "tessellation = gpd.read_file(\n",
+    "    momepy.datasets.get_path(\"bubenec\"), layer=\"tessellation\"\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Functions over classes\n",
+    "\n",
+    "The first key difference you may notice is that vast majority of functionality is now offered in a form of functions, rather than classes. Take an example fo equivalent rectangular index and its asssignemnt as a new column.\n",
```

Comment:
```suggestion
    "The first key difference you may notice is the vast majority of functionality is now offered as functions, rather than classes. Take equivalent rectangular index as an example and its assignment as a new column.\n",
```